src/components/track.tsx

```diff
@@ -3,49 +3,84 @@ import * as React from "react";
 import { accentColor } from "../constants";
 import { usePublicFileURL } from "../hooks/use-public-file-url";
 
+const TitleContainer = styled.div`
+  position: absolute;
+
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 4.5rem;
+  overflow: hidden;
+  transition: 0.5s ease;
+
+  background-color: rgba(0, 0, 0, 0.75);
+
+  -webkit-backdrop-filter: blur(10px);
+  border-top-right-radius: 3px;
+  border-bottom-right-radius: 3px;
+
+  z-index: 0;
+  cursor: default;
+`;
+
 const ImageContainer = styled.div`
   position: relative;
 
   display: block;
-  width: 100%;
   height: 15rem;
   overflow: hidden;
   margin-bottom: 1rem;
 
   border-radius: 5px;
 
   background-size: 100% auto;
-  background-position: center center;
   background-color: white;
+  background-position: center center;
+
+  &:hover ${TitleContainer} {
+    height: 100%;
+  }
 `;
 
-const Title = styled.h3`
-  position: absolute;
-  bottom: 1rem;
-  left: 0rem;
+const Title = styled.div`
+  color: white;
 
-  padding: 1rem 1.5rem;
+  padding-top: 1rem;
+  padding-left: 1rem;
```

Comment:
same here, why not have a unified padding?

Reply:
In this case unified padding would push Title and Sponsor too far apart.

Reply:
But we could have something like

```css
padding: 1rem;
padding-top: 0;
```

Reply:
`padding-bottom: 0` instead of `padding-top: 0` works, so I'll but that in